src/components/EpicView/EpicView.tsx

```diff
@@ -0,0 +1,132 @@
+import {Group, Stack, Text, Title, Flex, ScrollArea, TextInput, Box, Button, Center, Loader} from "@mantine/core";
+import {useNavigate} from "react-router-dom";
+import {useCanvasStore} from "../../lib/Store";
+import {useState} from "react";
+import {CreateIssueModal} from "../CreateIssue/CreateIssueModal";
+import {Issue, Sprint} from "../../../types";
+import {EpicWrapper} from "./EpicWrapper";
+import {useQuery} from "@tanstack/react-query";
+import {getEpics} from "./helpers/queryFetchers";
+
+
+export function EpicView() {
+  const navigate = useNavigate()
+  const projectName = useCanvasStore((state) => state.selectedProject?.name)
+  const [search, setSearch] = useState("")
+  const [createIssueModalOpened, setCreateIssueModalOpened] = useState(false)
+  const projectKey = useCanvasStore((state) => state.selectedProject?.key)
+  const [EpicWrappers, setEpicWrappers] = useState(
+      new Map<string, { issues: Issue[]}>()
+  )
+  const [searchedEpicWrappers, setSearchedepicWrappers] = useState(
+      new Map<string, { issues: Issue[]}>()
+  )
+
+  const updateEpicWrapper = (
+      key: string,
+      value: { issues: Issue[]}
+  ) => {
+      setEpicWrappers((map) => new Map(map.set(key, value)))
+      setSearchedepicWrappers((map) => new Map(map.set(key, value)))
+  }
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const currentSearch = event.currentTarget.value
+    setSearch(currentSearch)
+    //TODO implement search
+  }
+    const { isLoading: isLoadingBacklogIssues, isError: isErrorBacklogIssues } =
```

Comment:
The variable wording containing "backlog" seems somewhat off here, as we are only concerning epics in this list view.